php/php.editor/src/org/netbeans/modules/php/editor/model/nodes/ClassConstantDeclarationInfo.java

```diff
@@ -97,4 +96,61 @@ public PhpModifiers getAccessModifiers() {
         return PhpModifiers.fromBitMask(constantDeclaration.getModifier());
     }
 
+    @CheckForNull
+    protected static String getConstantValue(Expression expr) {
+        if (expr instanceof Scalar) {
+            return ((Scalar) expr).getStringValue();
+        }
+        if (expr instanceof UnaryOperation) {
+            UnaryOperation up = (UnaryOperation) expr;
+            if (up.getOperator() == UnaryOperation.Operator.MINUS
+                    && up.getExpression() instanceof Scalar) {
+                return "-" + ((Scalar) up.getExpression()).getStringValue(); //NOI18N
+            }
+        }
+        if (expr instanceof ArrayCreation) {
+            return getConstantValue((ArrayCreation) expr);
+        }
+        return null;
+    }
+
+    private static String getConstantValue(ArrayCreation expr) {
+        String debug = expr.toString();
+        StringBuilder sb = new StringBuilder("["); //NOI18N
+        Integer displayedElements = 0;
+        List<ArrayElement> elements = expr.getElements();
+        for (ArrayElement element : elements) {
+            if (displayedElements > 0) {
+                sb.append(", "); //NOI18N
+            }
+            sb.append(getConstantValue(element));
+            displayedElements++;
+            if (sb.length() > ARRAY_CUT_LENGTH) {
+                break;
+            }
+        }
+        if (displayedElements < elements.size()) {
+            sb.append(", ..."); //NOI18N
```

Comment:
Perhaps instead of `...` we could show something like `... (5 more elements)`?

Reply:
@KacerCZ @junichi11 

Let me know what you think about my comment. I am ready to merge. Thanks.

Reply:
I prefer to give more space for actual elements.
Maybe use shorter form `... (+5)` or `... (5 more)`.

Reply:
`... (5 more)` sounds good to me.

Reply:
Agree with @tmysik

Reply:
I made the change.

Reply:
Great, thanks!